### DISCUSSION TITLE
Integrare una caldaia ibrida tramite modbus

### DISCUSSION THREAD

**Discussion by Galmik** [indomus/forum#2531] [Hardware & Firmware]
Buongiorno a tutti, ho installato un sistema ibrido della Lamborghini caldaia + pompa di calore, come termostato ho un dispositivo proprietario wi-fi collegato in modbus alla caldaia che a sua volta colloquia in modbus con la pompa di calore, esiste un sistema per inserite Home assistan in questa catena e leggere i dati che si passano la caldaia con la pompa di calore e magari riuscire a gestire la caldaia con HA
Grazie

**Comment by Scialla** [comment]
https://www.home-assistant.io/integrations/modbus/

Cosa espone su protocollo modbus la tua caldaia devi saperlo tu o cercarlo in rete

**Reply by Galmik** [reply]
il mio problema non e tanto quello , ho già letto di qualcuno che ha interfacciato la caldaia con HA , ma non aveva il termostato wi-fi collegato sul bus, il mio problema e capire se ( visto che non ho conoscenza di modbus)  è possibile avere due master sulla stessa rete o se e possible intercettare ed interpretare i dai che master e slave si scambiano,
